appserver/tests/payara-samples/pom.xml

```diff
@@ -165,13 +160,8 @@
                     <plugin>
                         <artifactId>maven-surefire-plugin</artifactId>
                         <configuration>
-                            <environmentVariables>
```

Comment:
This may break Jenkins, I think it uses environment variables. Also there is a difference when you fork the JVM proces.

Reply:
haha, we control jenkins, so it's guaranteed to work!